packages/devtools_app/lib/src/flutter/scaffold.dart

```diff
@@ -328,31 +306,27 @@ class DevToolsScaffoldState extends State<DevToolsScaffold>
         onTap: _pushScreenToLocalPageRoute,
         tabs: [for (var screen in widget.tabs) screen.buildTab(context)],
       );
-      preferredSize = Tween<Size>(
-        begin: const Size.fromHeight(kToolbarHeight),
-        end: const Size.fromHeight(kToolbarHeight + 40.0),
-      ).evaluate(appBarCurve);
-      final animatedAlignment = Tween<Alignment>(
-        begin: Alignment.centerRight,
-        end: Alignment.bottomLeft,
-      ).evaluate(appBarCurve);
+      preferredSize = isNarrow
+          ? const Size.fromHeight(kToolbarHeight + 40.0)
+          : const Size.fromHeight(kToolbarHeight);
+      final animatedAlignment =
```

Comment:
rename to `alignment` since it is no longer animated?

Reply:
👍